insights/parsers/softnet_stat.py

```diff
@@ -8,42 +8,87 @@
 performed time_squeeze(third column), cpu collision i.e collision occur while obtaining
 device lock while transmitting cpu_collision packets (eighth column), received_rps
 number of times cpu woken up received_rps (ninth column), number of times reached flow
-limit count flow_limit_count (tenth column).
+limit count flow_limit_count (tenth column), backlog status (eleventh column),
+core id (twelfth column).
 
 The typical contents of ``cat /proc/net/softnet_stat`` file is as below::
 
-    00008e78 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
-    000040ee 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
-    0001608c 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
-    0000372f 00000001 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
+    00358fe3 00006283 00000000 00000000 00000000 00000000 00000000 00000000 00000000 000855fc 00000000 00000000 00000000
+    00953d1a 00000446 00000000 00000000 00000000 00000000 00000000 00000000 00000000 008eeb9a 00000000 00000000 00000001
+    02600138 00004bc7 00000000 00000000 00000000 00000000 00000000 00000000 00000000 02328493 00000000 00000000 00000002
+    02883c47 00007e2e 00000000 00000000 00000000 00000000 00000000 00000000 00000000 02280d49 00000000 00000000 00000003
+    01a35c9d 0002db94 00000001 00000000 00000000 00000000 00000000 00000000 00000000 008ee93a 00000000 00000000 00000004
 
     Column-01: packet_process: Packet processed by each CPU.
     Column-02: packet_drop: Packets dropped.
     Column-03: time_squeeze: net_rx_action.
-    Column-08: cpu_collision: collision occur while obtaining device lock while transmitting.
-    Column-09: received_rps: number of times cpu woken up received_rps.
-    Column-10: flow_limit_count: number of times reached flow limit count.
+    Column-09: cpu_collision: collision occur while obtaining device lock while transmitting.
+    Column-10: received_rps: number of times cpu woken up received_rps.
+    Column-11: flow_limit_count: number of times reached flow limit count.
+    Column-12: softnet_backlog_len: Backlog status
+    Column-13: index: core id owning this softnet_data
+
+For the proc file format details,
```

Comment:
The following docstring is not satisfied with the requirement of `sphinx`, and breaks the `doc-test`. Would you mind fixing them accordingly?
The insights-core follows the `Google Style Python Docstrings` please check [the examples here](https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google) for reference

Reply:
should be good now